source/agora/utils/PrettyPrinter.d

```diff
@@ -453,7 +460,9 @@ GCOQ...LRIJ(61,000,000), GCOQ...LRIJ(61,000,000), GCOQ...LRIJ(61,000,000),
 GCOQ...LRIJ(61,000,000), GCOQ...LRIJ(61,000,000), GCOQ...LRIJ(61,000,000),
 GCOQ...LRIJ(61,000,000), GCOQ...LRIJ(61,000,000)
 ====================================================
-Height: 1, Prev: 0x72e6...3b7d, Root: 0x07a8...acf4, Enrollments: [],
+Height: 1, Prev: 0x72e6...3b7d, Root: 0x07a8...acf4, Enrollments: []
+Signature: 0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000,
+Validators: [],
```

Comment:
It seems this one is different from the other ones `[]` vs `[0]`, even though it also has no signature?

Reply:
It is missing the validators field. I will add it.